apps/ledger-live-mobile/src/screens/SendFunds/04-Summary.tsx

```diff
@@ -141,15 +142,15 @@ function SendSummary({ navigation, route }: Props) {
     account.type === "Account" &&
     (account.subAccounts || []).some(subAccount => subAccount.balance.gt(0));
 
-  const onBuyEth = useCallback(() => {
+  const onBuy = useCallback(() => {
     navigation.navigate(NavigatorName.Exchange, {
       screen: ScreenName.ExchangeBuy,
       params: {
-        defaultAccountId: account?.id,
-        defaultCurrencyId: currencyOrToken?.id,
+        defaultAccountId: mainAccount.id,
+        defaultCurrencyId: mainAccount.currency.id,
```

Comment:
So using the main currency instead of the token currency? Just to confirm this is what we want?

Reply:
Yes, cf this comment bellow in the code -> https://github.com/LedgerHQ/ledger-live/pull/5069/files#diff-91b744818c341ff7f90d1dffdd6eece7372c0455b8c64285f254286e5e0c355bR264-R267

I updated the `onBuy` function here c0e00e0f69e0c4be181a41142a7837d871dd3ac0 to pass an account and thus avoid / reduce ambiguity